mu_repo/__init__.py

```diff
@@ -87,6 +109,11 @@ def main(config_file='.mu_repo', args=None, config=None):
         Print(msg)
         return Status(msg, False)
 
+    if config_file == '.mu_repo' and not os.path.isfile(config_file):
```

Comment:
Why limit config_file == '.mu_repo'? I think it should do that whenever config_file is not None (and then it should be passed as an argument to SearchConfigFile)... What do you think?

Another thing: the mu command may be used as a replacement for git in folders that contain a .git directory, so, in those cases, it should also not go upwards (and the SearchConfigFile should also stop going up if it finds the .git directory).